src/bindings/bnd_surface.cpp

```diff
@@ -44,6 +44,7 @@ void initSurfaceBindings(pybind11::module& m)
     .def("Degree", &BND_Surface::Degree, py::arg("direction"))
     .def("SpanCount", &BND_Surface::SpanCount, py::arg("direction"))
     .def("PointAt", &BND_Surface::PointAt, py::arg("u"), py::arg("v"))
+    .def("GetDomain", &BND_Surface::GetDomain, py::arg("direction"))
```

Comment:
This needs to be changed from "GetDomain" to "Domain". The names have to exactly match what we have in RhinoCommon so we can extract comments from the RhinoCommon source and apply them to the python/javascript versions